_posts/2023-06-12-changelog-196.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #196"
+date: 2023-06-12 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#10477] Add `unnecessary cast` inspection
+
+  {% include gif-img.html path="/assets/posts/changelog-196/unnecessary_cast" w="700px" %}
+
+* [#10524] Make `Run rustfmt` available as a pre-commit action
+
+* [#10514] Initial support for [E0505](https://doc.rust-lang.org/error_codes/E0505.html) using MIR framework
```

Comment:
Since it's available only under an experimental feature, let's move it to `internal` section or remove it entirely